Documentation/cmdref/cilium-agent.md

```diff
@@ -45,7 +45,7 @@ cilium-agent [flags]
       --datapath-mode string                          Datapath mode name (default "veth")
   -D, --debug                                         Enable debugging mode
       --debug-verbose strings                         List of enabled verbose debug groups
-  -d, --device string                                 Device facing cluster/external network for direct L3 (non-overlay mode) (default "undefined")
+  -d, --device strings                                List of devices facing cluster/external network for attaching bpf_netdev
```

Comment:
Could we let users also specify a device name regex? E.g. some nodes may have different device names than others when provisioning, but they might all share a same prefix for public interface e.g. `em*`, `eth*` or `pub*` (if configured as such) where the daemon would then internally gather the concrete list we need to attach to?

Reply:
Yeah, that's a good idea. I will implement it in a follow-up.